scripts/start_server_and_test

```diff
@@ -14,12 +14,14 @@ START_SERVER_SCRIPT="$1"
 PORT="$2"
 TEST_SCRIPT="$3"
 
-await_port --await-free "$PORT"
+await_port --await-free --timeout 120 "$PORT"
```

Comment:
address transient failures due to server not shutting down quickly enough prior to trying to spin it up again for next set of smoke test

Reply:
This is to prevent it from just spinning endlessly?

Does this mean your solution below doesn't reliably kill it every time?

Reply:
The kill works but it still takes time, since it's SIGTERM, not a SIGKILL. So sometimes 60 sec was not enough for the next web server _start_ command, which waits for the port to be free.